ipaserver/install/dns.py

```diff
@@ -108,6 +114,72 @@ def _disable_dnssec():
             conn.update_entry(entry)
 
 
+def _setup_dns_over_tls(options):
+    if os.path.isfile(paths.IPA_CA_CRT) and not options.dns_over_tls_cert:
+        # request certificate for DNS over TLS, using IPA CA
+        cert = paths.BIND_DNS_OVER_TLS_CRT
+        key = paths.BIND_DNS_OVER_TLS_KEY
+        certmonger.request_and_wait_for_cert(
+            certpath=(cert, key),
+            principal='DNS/%s@%s' % (FQDN, api.env.realm),
+            subject=str(DN(('CN', FQDN), default_subject_base(api.env.realm))),
+            storage="FILE"
+        )
+        constants.NAMED_USER.chown(cert, gid=constants.NAMED_GROUP.gid)
+        constants.NAMED_USER.chown(key, gid=constants.NAMED_GROUP.gid)
+
+    # setup and enable Unbound as resolver
+    forward_addrs = []
+    if options.dot_forwarders:
+        forward_addrs = ["forward-addr: %s" % fw for fw in options.dot_forwarders]
```

Comment:
@antoniotorresm , @abbra found another issue: "If I specify ipa-server-install --dns-over-tls --dns-policy=relaxed but didn't specify --dot-forwarder" then exception: TypeError: 'NoneType' object is not iterable

what about:
```
forward_addrs = ["forward-addr: %s" % fw for fw in options.dot_forwarders] if options.dot_forwarders else []
```